apps/test/tutorialExplorer/tutorialSetTest.js

```diff
@@ -0,0 +1,96 @@
+/** @file Tests for tutorialSet.js */
+
+var assert = require('assert');
+import TutorialSet from '@cdo/apps/tutorialExplorer/tutorialSet';
+var chai = require('chai');
+
+chai.should();
+
+describe("TutorialSet filterFn tests", function () {
+  let tutorials;
+
+  beforeEach(function () {
+    tutorials = [
```

Comment:
Is this an immutable static object? If so, I think we could replace line 10 with `const tutorials = [...]` and get rid of this beforeEach. If we're mutating tutorials, that won't work of course.